packages/headless/src/utils/highlight.ts

```diff
@@ -72,3 +101,43 @@ export function highlightString(params: HighlightParams): string {
   }
   return highlighted;
 }
+
+/**
+ * Highlight a suggestion with the given delimiters.
+ * @param suggestion The suggestion to highlight
+ * @param options The object contaning the delimiters used
+ */
+export function getHighlightedSuggestions(
```

Comment:
A better name for this function would be the singular, `highlightSuggestion` since it only accepts a single suggestion.
If we want to use the plural, then we should accept an array and loop.